package.json

```diff
@@ -7,20 +7,38 @@
     "@types/node": "^8.0.16",
     "@types/react": "^15.0.39",
     "@types/react-dom": "^15.5.1",
+    "eslint-config-prettier": "^2.3.0",
+    "husky": "^0.14.3",
```

Comment:
what does each one of these do and why are they "dependencies" and not "devDependencies"